neqo-transport/src/tracking.rs

```diff
@@ -559,6 +559,10 @@ impl RecvdPackets {
         }
     }
 
+    /// Length of the worst possible ACK frame, assuming only one range and ECN counts.
+    /// Note that this assumes one byte for the type and count of extra ranges.
+    pub const MAX_ACK_LEN: usize = 1 + 8 + 8 + 1 + 8 + 3 * 8;
```

Comment:
I can't think of a better name, but this isn't quite right.  What you have here is the longest length that an ACK can be if it contains a single range (and ECN counts).  A useful construct, for sure, because it is the safe minimum we'd need to be able to send a useful ACK.  So maybe:

```suggestion
    pub const USEFUL_ACK_LEN: usize = 1 + 8 + 8 + 1 + 8 + 3 * 8;
```